packages/@ourworldindata/utils/src/owidTypes.ts

```diff
@@ -1113,8 +1113,8 @@ export interface OwidGdocTag {
 
 // A minimal object containing metadata needed for rendering prominent links etc in the client
 export interface LinkedChart {
-    slug: string
-    path: string
```

Comment:
This implementation was actually flat out broken. I don't know why I went with such ambiguous names the first time 😵